doc/source/dev/how_to_release.rst

```diff
@@ -0,0 +1,104 @@
+Guidelines for Releasing PyWavelets
+===================================
+
+The following are guidelines for preparing a release of PyWavelets.  The
+notation vX.X.X in the commands below would be replaced by the actual release
+number.
+
+
+updating the release notes
+--------------------------
+Prior to the release, make sure the release notes are up to date.
+
+author lists can be generated via:
+
+```python  ./utils/gh_lists.py vP.P.P..```
+(where vP.P.P is the previous release number)
+
+and the lists of Issues closed and PRs merged via:
+
+```python ./utils/gh_lists.py vX.X.X```  (script requires Python 2.X to run)
+
+
+Tag the release and trigger the build of Windows wheels
+-------------------------------------------------------
+
+Change ISRELEASED to TRUE in `setup.py` and commit.
+
+Appveyor will build wheels for windows.  If the commit with ISRELEASED=True
+is submitted as a PR, the wheels can be downloaded from appveyor once it has
+run on the PR.  They can be found under the "Artifacts" tab in the Appveyor
+interface.  These should be downloaded so that they can later be uploaded to
+pypi.python.org.
+
+
+Tag the release via:
+
+```git tag -s vX.X.X```
+
+Then push the vX.X.X tag to master.
+
+
+Create the source distribution
+------------------------------
+
+Remove untracked files and directories via git clean.
+**Warning:**  This will delete files & directories that are not under version
+control so you may want to do a dry run first by adding -n, so you can see what
+will be removed:
+
+```git clean -xfdn```
+
+Then run without -n:
+
+```git clean -xfd```
+
+Create the source distribution files via:
+
+```python setup.py sdist --formats=gztar,zip```
+
+
+Upload the release to pypi
+--------------------------
+
+The source distributions created above can be securely uploaded to
+pypi.python.org using twine:
+
+```twine upload -s dist/*```
+
+Also manually upload the Windows wheels (see above) through the pypi web
+interface.
```

Comment:
you can also do this with twine. maybe best to first download the Windows wheels into ``dist/`` and then the single run of ``twine upload -s dist/`` will do it all at once.